seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/source/split/FileSourceSplitEnumerator.java

```diff
@@ -123,8 +123,10 @@ public int currentUnassignedSplitSize() {
 
     @Override
     public void registerReader(int subtaskId) {
+        Set<Integer> readers = context.registeredReaders();
```

Comment:
Why change it?

Reply:
> Why change it?

I just reconfirmed that there is no such problem,  I have restored it